main.js

```diff
@@ -1284,23 +1284,22 @@ function main() {
     });
 
     zwave.on('value removed', function (nodeID, comClass, instance, index) {
-        adapter.log.info('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
+        adapter.log.debug('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
 
-        deleteDevice(nodeID, function() { //why here and not on "node removed" ?
```

Comment:
Ahh, I see. I had this question before. Good to know that deleting the device here is indeed not the desired behavior.

Reply:
Well the node is refreshed and reported as new. But the value changes are reported individually by the openzwave-shared module. That's why you see tons of delete entries in the bug report.
The change from 1.4 to 1.6 is that it actually **does** refresh as the openzwave cache is dropped.

Reply:
To add to that: what I could not test so far if it also works when that value removed is fired without refreshing so when the values change "normally". I simply don't have an idea about a use case for it.